platform-workflow-module/src/main/java/com/softicar/platform/workflow/module/workflow/user/configuration/AGWorkflowUserConfigurationTable.java

```diff
@@ -51,8 +50,8 @@ public void customizeLoggers(EmfChangeLoggerSet<AGWorkflowUserConfiguration> log
 	public void customizeAttributeProperties(IEmfAttributeList<AGWorkflowUserConfiguration> attributes) {
 
 		attributes//
-			.editAttribute(AGWorkflowUserConfiguration.SUBSTITUTE)
-			.setInputFactory(UserInput::new);
```

Comment:
Can we remove the `UserInput` class?
It seems orphaned now, and a bit useless in general

Reply:
unfortunately, we cannot, it is use in EAS aside from EMF-attributes